queries.yaml

```diff
@@ -616,6 +616,15 @@ post_process:
         - "id:left"
         - "id:right"
 
+  # drop any name on locality boundaries at zooms 11 and 12 (min two zooms).
+  - fn: vectordatasource.transform.drop_names
```

Comment:
nit: consider adding a note here saying that the data comes in at zoom 11 in the boundaries.yaml and changes should be coordinated, and add note over there as well?

Reply:
Cool, good idea. Added in 7c0a1d8.

(Alternatively, we could just have the `start_zoom` here be 0, and then it wouldn't matter if we changed the locality `min_zoom` to 10... but that means running this filter at 10 zoom levels where there isn't any data for it to process, which seems wasteful - so instead just sticking with pointing the problem out and requiring human attention to make sure it doesn't get out of sync.)